net-p2p/go-ipfs-bin/go-ipfs-bin-0.4.14.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1
+
+DESCRIPTION="go-ipfs is the main implementation of IPFS."
```

Comment:
`DESCRIPTION` should not contain a package name.

Reply:
@Zlogene, now it's not a proper sentence, so it shouldn't end with a full stop.